doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1024,11 +1024,6 @@ the next command fails because p does not bind in the instance of n.
 
    Fail Check (exists_different p).
 
-.. rocqtop:: in
-
-   Notation "[> a , .. , b <]" :=
-     (cons a .. (cons b nil) .., cons b .. (cons a nil) ..).
-
```

Comment:
I believe this example was a leftover from some refactoring of "Notations with recursive patterns" below. Not sure if we want to reintegrate it somewhere.